ui/src/admin/containers/AdminInfluxDBPage.js

```diff
@@ -245,7 +245,7 @@ class AdminInfluxDBPage extends Component {
 
 const {arrayOf, func, shape, string} = PropTypes
 
-AdminInfluxDBPage.propTypes = {
+DisconnectedAdminInfluxDBPage.propTypes = {
```

Comment:
I do not take issue, but I am curious -- why prefix with `Disconnected`? Does that suggest the component needs to be wrapped with `connect`?

Reply:
Yep. It's easier to test the component without wiring it up to redux via `connect`.

In other contexts (e.g. the redux docs) I've seen the convention of prefixing the `connect`ed version of a component with either `Visible`, `Connected`, `Wrapper` or similar—so `connect(Foo)` would be called `VisibleFoo` or whatever. It was too much of a hassle in this instance so I adopted the somewhat awkward `Disconnected` prefix.